### PR TITLE
Prevent scroll for subview was zoomed.

### PR DESCRIPTION
**Pull request by YKW93**
Prevent scroll for subview was zoomed.

use case below.
```java
carouselView.setOnScrollChangeListener(new View.OnScrollChangeListener() {
            @Override
            public void onScrollChange(View v, int scrollX, int scrollY, int oldScrollX, int oldScrollY) {
                int dx = oldScrollX - scrollX;
                if (v instanceof ZoomableImageView) {
                    boolean scrollable = ((ZoomableImageView) v).canScroll(-dx);
                    carouselView.setLockScroll(!scrollable);
                }
            }
        });
```

### CODE REVIEW

**Comment by ablyeom** [comment]
@sayyam I need this function, please merge it.

**Comment by HansenBudinata** [comment]
@sayyam I need this function as well, please kindly merge it =D

File: carouselview/src/main/java/com/synnapps/carouselview/CarouselView.java [hunk 1]
```diff
@@ -242,15 +248,16 @@ private void setDisableAutoPlayOnUserInteraction(boolean disableAutoPlayOnUserIn
     private void setData() {
         CarouselPagerAdapter carouselPagerAdapter = new CarouselPagerAdapter(getContext());
         containerViewPager.setAdapter(carouselPagerAdapter);
-        mIndicator.setViewPager(containerViewPager);
-        mIndicator.requestLayout();
-        mIndicator.invalidate();
-        containerViewPager.setOffscreenPageLimit(getPageCount());
-        playCarousel();
+        if(getPageCount() > 1) {
```
Comment (sayyam) [inline carouselview/src/main/java/com/synnapps/carouselview/CarouselView.java]: +1

Comment (ablyeom) [reply carouselview/src/main/java/com/synnapps/carouselview/CarouselView.java:251]: Can you release this version? I need this function

**Pull request merged**